webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/Taxonomies.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormGroup,
+  FormSelectOption,
+  FormSelect,
+} from '@patternfly/react-core';
+
+import { translate as __ } from '../../../../../common/I18n';
+
+const Taxonomies = ({
+  organizationId,
+  organizations,
+  handleOrganization,
+  locationId,
+  locations,
+  handleLocation,
+  isLoading,
+}) => (
+  <>
+    <FormGroup label="Organization" isRequired fieldId="reg_organization">
+      <FormSelect
+        value={organizationId}
+        onChange={v => handleOrganization(v)}
+        className="without_select2"
+        id="reg_organization_select"
+        isDisabled={isLoading}
+        isRequired
+      >
+        <FormSelectOption value="" label={__('Any Organization')} />
```

Comment:
Could the value be rather 'Not specified'? Also the inline help could say, "If no Organization is set, the default Organization of the user is assumed."

Likewise for the locations.